### PR TITLE
Avoid hard code in codebase

### PR DESCRIPTION
**Pull request by fisherxu**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
Avoid hard code

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/cc @sids-b @rohitsardesai83

### CODE REVIEW

File: cloud/pkg/cloudhub/servers/wsserver/server.go [hunk 1]
```diff
@@ -15,11 +15,12 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/handler"
 
 	"github.com/gorilla/mux"
+	"github.com/kubeedge/kubeedge/cloud/pkg/cloudhub/common/model"
```
Comment (sids-b) [inline cloud/pkg/cloudhub/servers/wsserver/server.go]: please seperate vendor and kubeedge packages.

Comment (fisherxu) [reply cloud/pkg/cloudhub/servers/wsserver/server.go]: Done.

**Pull request merged**